File: src/main/java/com/unicorn/config/WebConfig.java
package com.unicorn.config;

import org.springframework.boot.autoconfigure.AutoConfigureAfter;
import org.springframework.boot.autoconfigure.web.DispatcherServletAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

/**
 * Created by dtimchenko on 02-Aug-17.
 */

@Configuration
@EnableWebMvc
@ComponentScan(basePackages = {"com.unicorn.controller", "com.unicorn.service"})
@AutoConfigureAfter(DispatcherServletAutoConfiguration.class)
public class WebConfig extends WebMvcConfigurerAdapter {

    @Override
    public void addResourceHandlers(ResourceHandlerRegistry registry) {
        registry.addResourceHandler("/resources/**")
                .addResourceLocations("/resources/", "file:/resources/");
        super.addResourceHandlers(registry);
    }
}

File: src/main/java/com/unicorn/restemulator/config/WebConfig.kt
package com.unicorn.restemulator.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.EnableWebMvc

/**
 * Created by dtimchenko on 10-Aug-17.
 */

@Configuration
@EnableWebMvc
@ComponentScan(basePackages = arrayOf("com.unicorn.restemulator.controller"))
open class WebConfig